keras/layers/recurrent.py

```diff
@@ -217,7 +218,15 @@ def call(self, x, mask=None):
                                 'argument, including the time axis. '
                                 'Found input shape at layer ' + self.name +
                                 ': ' + str(input_shape))
-        if self.stateful:
+        if self.initial_states is not None:
+            if self.stateful:
+                raise Exception('Keras does not currently support stateful Recurrent '
+                                'layers with custom initial states.')
+            if type(self.initial_states) in (list, tuple):
+                initial_states = self.initial_states
+            else:
+                initial_states = [self.initial_states]
```

Comment:
I mean a symbolic tensor (e.g. TF or Theano tensor) as opposed to a numeric tensor (a Numpy array).

I don't think that most people would want to initialize the states via symbolic tensors. What is your use case?